.devcontainer/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.154.0/containers/java/.devcontainer/base.Dockerfile
+
+# [Choice] Java version: 11, 15
+ARG VARIANT="15"
```

Comment:
Why is 15 an option? I think we are going only with 11, don't we?

Reply:
+1